lib/js_routes.rb

```diff
@@ -36,6 +36,7 @@ class JsRoutes
 
   LAST_OPTIONS_KEY = "options".freeze
   FILTERED_DEFAULT_PARTS = [:controller, :action, :subdomain]
+  URL_OPTIONS = [:protocol, :domain, :host, :port]
```

Comment:
Why isn't `subdomain` in this hash? Why is it processed differently from `domain`?